tests/unit/customizations/test_commands.py

```diff
@@ -121,8 +121,8 @@ def test_expected_events_are_emitted_in_order(self):
         driver.main('s3 foo'.split())
 
         self.assert_events_fired_in_order([
-            'building-command-table.main',
             'building-top-level-params',
+            'building-command-table.main',
```

Comment:
I had to change the order of these events because the global arg table must be built as part of the `_build_builtin_commands` method which gets run before the session emits a `building-command-table.main` event.